src/rules/javascript/index.js

```diff
@@ -5,4 +5,5 @@ export default {
   mozindexeddb_property: ESLINT_WARNING,
   opendialog_remote_uri: ESLINT_WARNING,
   opendialog_nonlit_uri: ESLINT_WARNING,
+  eval_string_arg: ESLINT_ERROR,
```

Comment:
This is fine for now - but I'm completely unclear with this rule what message type it is. We should see what the current validator makes of this to know.